config/toml.go

```diff
@@ -285,6 +285,31 @@ enabled = {{ .GRPC.BlockService.Enabled }}
 [grpc.block_results_service]
 enabled = {{ .GRPC.BlockResultsService.Enabled }}
 
+#
+# Configuration for privileged gRPC endpoints, which should **never** be exposed
+# to the public internet.
+#
+[grpc.privileged]
+# The host/port on which to expose privileged gRPC endpoints.
+laddr = "{{ .GRPC.Privileged.ListenAddress }}"
+
+#
+# Configuration specifically for the gRPC pruning service, which is considered a
+# privileged service.
+#
+[grpc.privileged.pruning_service]
+
+# Only controls whether the pruning service is accessible via the gRPC API - not
+# whether a previously set pruning service retain height is honored by the
+# node.
+#
+# To disable the influence of previously set pruning service retain height(s) on
+# node pruning, this endpoint should be enabled and the relevant pruning service
+# retain heights should be set to 0.
```

Comment:
I am not 100% sure I understand this comment. I guess what you want to say is that one should try to set the retain height using this gRPC endpoint if he wants a different height to be considered? Because if one enables the companion and tries to set the height to 0 when it was previously set to some thing >0 by either the app or another data companion, this operation will fail.

Reply:
Hmm, I just copied this from the ADR-101 document. @thanethomson can we make this more clear?

Reply:
I've removed this from the ADR now. It's no longer relevant.